src/UKHO.ERPFacade.Common/Configuration/ISapConfiguration.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace UKHO.ERPFacade.Common.Configuration
+{
+    public interface ISapConfiguration
+    {
+        public string BaseAddress { get; set; }
+
+        public string SapServiceOperation { get; set; }
+
+        public string Username { get; set; }
```

Comment:
```suggestion
        public string UserName { get; set; }
```

Reply:
This is added as per keyvault configuration for SAP